system-test/spanner.ts

```diff
@@ -1050,23 +1369,45 @@ describe('Spanner', () => {
 
   describe('Databases', () => {
     const TABLE_NAME = 'SingersTest';
-    it('should auto create a database', done => {
-      const database = instance.database(generateName('database'));
+    const autoCreateDatabase = (done, databaseId) => {
+      const database = instance.database(generateName(databaseId));
 
       database.get({autoCreate: true} as GetDatabaseConfig, err => {
         assert.ifError(err);
         RESOURCES_TO_CLEAN.push(database);
         database.getMetadata(done);
```

Comment:
nit: Maybe add a check here to verify that the database was created with the correct dialect.

Reply:
Adding thanks